client/src/test/java/io/pravega/client/stream/impl/IdempotentEventStreamWriterTest.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
```

Comment:
This license header is not the one we are supposed to use, check the `HEADER` file in the project root.